client/state/comments/actions.js

```diff
@@ -122,7 +133,8 @@ export function requestPostComments( siteId, postId, status = 'approved' ) {
 		const requestId = createRequestId( siteId, postId, query );
 
 		// if the request status is in-flight or completed successfully, no need to re-fetch it
-		if ( postCommentRequests && [ COMMENTS_REQUEST, COMMENTS_REQUEST_SUCCESS ].indexOf( postCommentRequests.get( requestId ) ) !== -1 ) {
+		if ( postCommentRequests &&
+			[ COMMENTS_REQUEST, COMMENTS_REQUEST_SUCCESS ].indexOf( get( postCommentRequests, [ requestId ] ) ) !== -1 ) {
```

Comment:
@blowery thanks for the review: what's different the way this is transformed? it looks to me like it's preserving the existing behavior - I must be missing something…